src/app/errors/reducer.ts

```diff
@@ -2,12 +2,18 @@ import { Reducer } from 'redux';
 import { getType } from 'typesafe-actions';
 import * as navigation from '../navigation';
 import { AnyAction } from '../types';
+import { clearCurrentError, recordError } from './actions';
 import { notFound } from './routes';
 import { State } from './types';
+
 export const initialState = {};
 
 const reducer: Reducer<State, AnyAction> = (state = {}, action) => {
   switch (action.type) {
+    case getType(clearCurrentError):
+      return { ...state, error: undefined };
```

Comment:
you could also `return initialState` here, not sure if that would be more correct or not